app/src/main/java/com/example/findissues/ui/fragments/HomeFragment.kt

```diff
@@ -85,39 +98,51 @@ class HomeFragment : Fragment() {
             pinnedRepoAdapter.setUpPinnedRepoList(it)
         }
 
-        with(binding) {
-            tvFollowers.setOnClickListener {
-                setUpFragment(FollowersFragment())
-            }
-            tvFollowing.setOnClickListener {
-                setUpFragment(FollowingFragment())
-            }
-            tvRepositories.setOnClickListener {
-                setUpFragment(RepositoryFragment())
-            }
-            tvStarred.setOnClickListener {
-                setUpFragment(StarredFragment())
+//        with(binding) {
+//            tvFollowers.setOnClickListener {
+//                setUpFragment(FollowersFragment())
+//            }
+//            tvFollowing.setOnClickListener {
+//                setUpFragment(FollowingFragment())
+//            }
+//            tvRepositories.setOnClickListener {
+//                setUpFragment(RepositoryFragment())
+//            }
+//            tvStarred.setOnClickListener {
+//                setUpFragment(StarredFragment())
+//            }
+//        }
+
+//        return binding.root
+        return ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.Default)
```

Comment:
change composition strategy to [DisposeOnViewTreeLifecycleDestroyed](https://developer.android.com/reference/kotlin/androidx/compose/ui/platform/ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed). Read more about ViewCompositionStrategy [here](https://medium.com/androiddevelopers/viewcompositionstrategy-demystefied-276427152f34)